client/sites/overview/components/migration-overview/components/migration-started-difm.tsx

```diff
@@ -5,12 +5,16 @@ import type { SiteDetails } from '@automattic/data-stores';
 
 export const MigrationStartedDIFM = ( { site }: { site?: SiteDetails } ) => {
 	const translate = useTranslate();
+	const migrationSourceSiteDomain = site?.options?.migration_source_site_domain
+		? site?.options?.migration_source_site_domain?.replace( /^https?:\/\/|\/+$/g, '' )
+		: translate( 'your site' );
+
 	const title = translate( 'Your migration is underway' );
 	const subTitle = translate(
-		"Sit back as {{strong}}%(siteName)s{{/strong}} transfers to its new home. Here's what you can expect.",
+		"Sit back as {{strong}}%(siteUrl)s{{/strong}} transfers to its new home. Here's what you can expect.",
```

Comment:
this change will make the string not recognized by the translation function. We can do one of these things:
1. stick to `siteName`, kind of ugly but will work right away. We should make a follow up PR with the string freeze label so it can be merged once translation is done
1. Wait for translation
1. Add a conditional to check whether the new string is ready.

I lean towards 1. , but let me know what you think